py/selenium/webdriver/remote/remote_connection.py

```diff
@@ -471,6 +471,9 @@ def _request(self, method, url, body=None):
             request.add_header('Accept', 'application/json')
             request.add_header('Content-Type', 'application/json;charset=UTF-8')
 
+            base64string = base64.b64encode('%s:%s' % (parsed_url.username, parsed_url.password))
```

Comment:
It doesn't seem to break the driver, but do we always want to pass an auth header in all of the requests?